#Fix for 14499 to calendar
#Cherry Pick
=svnm= https://source.sakaiproject.org/svn/calendar/trunk@52607 calendar 

#Specific version specified

#Fill in the SAK or something unique, leave it like this if theres no real patch
--- /dev/null
+++ SAK-14499.tmp
@@ -0,0 +1 @@
+ 
